spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 ENV["RACK_ENV"] = "test"
 
-require 'setup_test_database'
+require "setup_test_database"
+require "../db_test_connection_helper"
```

Comment:
I needed to change this to:

require_relative "../lib/db_test_connection_helper"

Otherwise it threw an error - does it still work for you guys if you change it to this? Don't know why it's being temperamental

Reply:
same